demos/eng-2019/src/convert_to_csv.py

```diff
@@ -22,7 +23,11 @@ def convert_to_csv(args):
 
     segmentations = dict()
     for cell_id, cell_obj in cells_json.items():
-        segmentations[cell_id] = to_diamond(cell_obj['xy'][0], cell_obj['xy'][1], 50)
+        segmentations[cell_id] = to_diamond(cell_obj['xy'][0], cell_obj['xy'][1], 50).tolist()
+
+    # Pandas does not make intermediate paths
+    output_dir = Path(args.output_cells).parent
+    output_dir.mkdir(parents=True, exist_ok=True)
```

Comment:
This is correct that `pandas` does not make intermediate paths, but `snakemake` should (at least it used to). I wonder if it changed

Reply:
I also assumed it would